### YamlMime:ManagedReference
items:
- uid: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException
  id: AddInSegmentDirectoryNotFoundException
  children:
  - System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor
  - System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor(System.String)
  - System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: AddInSegmentDirectoryNotFoundException
  nameWithType: AddInSegmentDirectoryNotFoundException
  fullName: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException
  type: Class
  summary: "A kivétel fordul elő, amikor egy szegmens directory hiányzik a feldolgozási sor könyvtárstruktúrát."
  remarks: "Bővítmény aktiválódik, amikor a folyamat a gazdagép összeállított. A rendszer megköveteli az, hogy a feldolgozási sori szegmenseket kerüljenek-e egy megadott könyvtárstruktúrát a megadott könyvtár nevét, a [csővezeték fejlesztési követelmények](http://msdn.microsoft.com/en-us/ef9fa986-e80b-43e1-868b-247f4c1d9da5)."
  syntax:
    content: 'public class AddInSegmentDirectoryNotFoundException : Exception'
  inheritance:
  - System.Exception
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor
  id: '#ctor'
  parent: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException
  langs:
  - csharp
  name: AddInSegmentDirectoryNotFoundException()
  nameWithType: AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException()
  fullName: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException()
  type: Constructor
  assemblies:
  - System.AddIn
  namespace: System.AddIn.Hosting
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException&quot;> </xref> osztály."
  remarks: "Ez a konstruktor inicializálja a <xref:System.Exception.Message%2A>tulajdonság, amely leírja a hibát (például &quot;a célként megadott alkalmazástartományt eltávolították a memóriából.&quot;) rendszer által megadott üzenetet az új példányának.</xref:System.Exception.Message%2A> Ezt az üzenetet a rendszer aktuális kulturális környezetének figyelembe veszi.       Az alábbi táblázat a kezdeti tulajdonságértékek <xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException>osztály</xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException> példánya      | Tulajdonság |} Érték |}   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Egy null hivatkozás (`Nothing` a Visual Basic nyelvben). |}   | <xref:System.Exception.Message%2A>| Az üzenet honosított hibakarakterlánc. |}</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public AddInSegmentDirectoryNotFoundException ();
    parameters: []
  overload: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException
  langs:
  - csharp
  name: AddInSegmentDirectoryNotFoundException(String)
  nameWithType: AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(String)
  fullName: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(String)
  type: Constructor
  assemblies:
  - System.AddIn
  namespace: System.AddIn.Hosting
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException&quot;> </xref> a megadott üzenet osztályt."
  remarks: "Az alábbi táblázat a kezdeti tulajdonságértékek <xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException>osztály</xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException> példánya      | Tulajdonság |} Érték |}   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Egy null hivatkozás (`Nothing` a Visual Basic nyelvben). |}   | <xref:System.Exception.Message%2A>| Az üzenet honosított hibakarakterlánc. |}</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public AddInSegmentDirectoryNotFoundException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Egy üzenet, amely leírja a hibát."
  overload: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException
  langs:
  - csharp
  name: AddInSegmentDirectoryNotFoundException(SerializationInfo,StreamingContext)
  nameWithType: AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(SerializationInfo,StreamingContext)
  fullName: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.AddIn
  namespace: System.AddIn.Hosting
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException&quot;> </xref> a szerializálási adatok osztályt."
  remarks: "Ez a konstruktor nevezik meglévő kölcsönökre adatfolyam során átvitt kivétel objektum deszerializálása közben. További információkért lásd: [XML és a SOAP-szerializálási](~/add/includes/ajax-current-ext-md.md).       Az alábbi táblázat a kezdeti tulajdonságértékek <xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException>osztály</xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException> példánya      | Tulajdonság |} Érték |}   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Egy null hivatkozás (`Nothing` a Visual Basic nyelvben). |}   | <xref:System.Exception.Message%2A>| Az üzenet honosított hibakarakterlánc. |}</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: protected AddInSegmentDirectoryNotFoundException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "A szerializált objektum adatokat tartalmazó objektum."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "A cél- és objektum adatok környezetfüggő információit."
  overload: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException
  langs:
  - csharp
  name: AddInSegmentDirectoryNotFoundException(String,Exception)
  nameWithType: AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(String,Exception)
  fullName: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(String,Exception)
  type: Constructor
  assemblies:
  - System.AddIn
  namespace: System.AddIn.Hosting
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException&quot;> </xref> megadott hibaüzenet, és hivatkozni kell a belső kivételben találhatók okozza-e ezt a kivételt, a osztályt."
  remarks: "Kivételt vált ki, mert egy előző kivétel közvetlen eredménye tartalmaznia kell egy hivatkozást az előző kivétel az a <xref:System.Exception.InnerException%2A>tulajdonság.</xref:System.Exception.InnerException%2A> A <xref:System.Exception.InnerException%2A>tulajdonság a konstruktorban, vagy null hivatkozás. Ha átadott ugyanazt az értéket adja vissza a <xref:System.Exception.InnerException%2A>tulajdonság nem adja meg a belső kivétel értékét a konstruktor.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       Az alábbi táblázat a kezdeti tulajdonságértékek <xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException>osztály</xref:System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException> példánya      | Tulajdonság |} Érték |}   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| A belső kivétel referencia. |}   | <xref:System.Exception.Message%2A>| Az üzenet honosított hibakarakterlánc. |}</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public AddInSegmentDirectoryNotFoundException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "Egy üzenet, amely leírja a hibát."
    - id: innerException
      type: System.Exception
      description: "A kivétellel, hogy a jelenlegi kivétel okozza. Ha a `innerException` paraméter nem egy null hivatkozás, a jelenlegi kivétel következik be egy catch blokkban, amely kezeli a belső kivételben találhatók."
  overload: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor
  parent: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException
  isExternal: false
  name: AddInSegmentDirectoryNotFoundException()
  nameWithType: AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException()
  fullName: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException()
- uid: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor(System.String)
  parent: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException
  isExternal: false
  name: AddInSegmentDirectoryNotFoundException(String)
  nameWithType: AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(String)
  fullName: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException
  isExternal: false
  name: AddInSegmentDirectoryNotFoundException(SerializationInfo,StreamingContext)
  nameWithType: AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(SerializationInfo,StreamingContext)
  fullName: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor(System.String,System.Exception)
  parent: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException
  isExternal: false
  name: AddInSegmentDirectoryNotFoundException(String,Exception)
  nameWithType: AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(String,Exception)
  fullName: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException(String,Exception)
- uid: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException.#ctor*
  parent: System.AddIn.Hosting.AddInSegmentDirectoryNotFoundException
  isExternal: false
  name: AddInSegmentDirectoryNotFoundException
  nameWithType: AddInSegmentDirectoryNotFoundException.AddInSegmentDirectoryNotFoundException
